configure.ac

```diff
@@ -3321,6 +3321,15 @@ AC_ARG_WITH([libmongoc],
 SAVE_CPPFLAGS="$CPPFLAGS"
 SAVE_LDFLAGS="$LDFLAGS"
 
+if test "x$with_libmongoc" = "xyes"
```

Comment:
Please use the same style as the rest of configure.ac, so `if; then` on the same line, and indent with 2 spaces.